docs/guides/advanced/images.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Images & files
+---
+
+# Working with images and files
+
+You can upload images and files to a Coda doc, including in a table using dedicated column types. Packs likewise have the ability to work with binary data.
+
+[View Sample Code][samples]{ .md-button }
+
+
+## Parameters and result types
+
+Formulas can accept an image as a parameter using the parameter type `Image`, which is then passed to the `execute` function as a URL. It currently isn't possible to pass non-image files however. See the [Parameters guide][parameters_images] for more information.
+
+Likewise, formulas can return images and files as URLs, using the hint types `ImageReference`, `ImageAttachment`, or `Attachment`. See the [Data types guide][data_types_images] for more information.
```

Comment:
Even if the details are documented elsewhere, I think it might be useful to put a summary of the different hint types here, and in particular highlight that ImageAttachment is almost always the right answer and ImageReference is discouraged unless you have a specific reason to hotlink.

Reply:
Done